lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -621,6 +621,20 @@ SPIRVType *LLVMToSPIRVBase::transPointerType(SPIRVType *ET, unsigned AddrSpc) {
 // simply not true.
 SPIRVType *LLVMToSPIRVBase::transSPIRVJointMatrixINTELType(
     SmallVector<std::string, 8> Postfixes) {
+  auto ParseInteger = [this](StringRef Postfix) -> ConstantInt * {
+    unsigned long long N = 0;
+    if (consumeUnsignedInteger(Postfix, 10, N)) {
+      BM->getErrorLog().checkError(
```

Comment:
Can't comment the description above the function.
I guess we need to update the LLVM IR representation description as there are less arguments than in the added test:
```
// Representation in LLVM IR before the translator is a pointer to an opaque
// structure:
// %spirv.JointMatrixINTEL._%element_type%_%rows%_%cols%_%scope%_%use%
```

Reply:
The idea was that %layout% will be removed (bot now, since it's a breaking change). But I agree - let's return it here and update the description once we are ready.